### PR TITLE
UPSTREAM: <carry>: openshift: Enable printcolumn for phase and move status to priority

### PR DESCRIPTION
**Pull request by enxebre**
Enable printcolumn for phase and move status to priority

### CODE REVIEW

**Comment by enxebre** [comment]
/hold
until we merge https://github.com/openshift/cluster-api-provider-aws/pull/261

**Review by bison — APPROVED** [review]
/lgtm

**Pull request merged**
